### ISSUE TITLE
autopep8 command not found

### ISSUE THREAD

**Issue by snowleung** [paetzke/py-autopep8.el#13]
It's work well at emacs24.4 (GNU Emacs 24.4.1 (x86_64-apple-darwin13.4.0))
but 
when I use Emacs of Mac OS X(This is GNU Emacs 24.5.1 (x86_64-apple-darwin13.4.0, NS apple-appkit-1265.21),with GUI, display "autopep8 command not found" on the mini buffer.
also, 
I use this M-x py-autopep8-buffer to call the function.
thanks.

**Comment by paetzke** [comment]
Could you try this:

```
M-x shell-command RET
autopep8 RET
```
What happens when you do this?

**Comment by snowleung** [comment]
![7ac2f992-d1ab-4221-a1ba-c214029c221f](https://cloud.githubusercontent.com/assets/1115526/9351408/aa5e71e0-468b-11e5-83d0-c82e2c6ea125.png)

**Comment by flexgrip** [comment]
Same exact thing here. Works totally fine in my terminal. But this emacs plugin does not work.

**Comment by mcobzarenco** [comment]
Same here +1

**Comment by mcobzarenco** [comment]
Right, the problem in my case was that the `PATH` variable was set differently in Emacs compared to the shell. @flexgrip @snowleung try:
```
M-x shell-command RET
echo $PATH RET
```
and compare it with `echo $PATH` in your shell.

**Comment by snowleung** [comment]
@mcobzarenco hey, you are right, they're different.
in terminal, it seems include some ... ruby library?

**Comment by wbolster** [comment]
try `exec-path-from-shell`.


this issue can be closed as it's not a bug in this package.

**Comment by ideasman42** [comment]
Melpa now points to a fork of this package maintained here: https://github.com/ideasman42/emacs-py-autopep8

This adds `py-autopep8-command` (defaulting to `autopep8`). So you can point to the location of autopep8 without it being in your path.
